pkg/apis/playlist/v1/handler.go

```diff
@@ -2,55 +2,70 @@ package v1
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/grafana/grafana/pkg/apis"
```

Comment:
since @mildwonkey is out today, i'm going to proxy a ಠ_ಠ at the order of the imports

Reply:
i'm so proud 😭